frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -210,11 +210,11 @@ export default function AppLayout() {
   return (
     <>
       <div className="font-sans min-h-screen w-full flex flex-col">
-        <div className="flex-1 h-full grid md:grid-cols-[220px_1fr] lg:grid-cols-[280px_1fr]">
+        <div className="flex-1 h-full grid md:grid-cols-[280px_1fr]">
```

Comment:
do we need to use a hardcoded pixel measurement?

Reply:
We could use the `w-72` class potentially. I'll update it :+1: